### PR TITLE
Fixes configuration for Azure integrationt tests

### PR DESCRIPTION
**Pull request by dmccown**
Updates some calls in train so we can access the configuration options
for some integration tests.

Signed-off-by: David McCown <dmccown@chef.io>

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -162,7 +162,9 @@ namespace :test do
       sh("cd #{integration_dir}/build/ && terraform workspace new #{tf_workspace}")
 
       # Generate Azure crendentials
-      creds = Train.create('azure').connection.connect
+      connection = Train.create('azure').connection
+      connection.connect
```
Comment (TrevorBramble) [inline Rakefile]: That shouldn't be necessary? https://github.com/chef/train/blob/a931a4dccf8d01b0a99de28d5c17b2b484e6ce06/lib/train/transports/azure.rb#L50

Comment (dmccown) [reply Rakefile]: Ahh nice catch. I'll update.

Comment (dmccown) [reply Rakefile]: I updated and confirmed it works without the explicit connect.

**Review by TrevorBramble — APPROVED** [review]
Awesome!

**Review by jquick — APPROVED** [review]
Thanks @dmccown

**Pull request merged**